CRITICAL - 2024-01-14 04:12:20 --> Class "App\Controllers\Date" not found
in APPPATH/Controllers/User.php on line 56.
 1 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 2 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 3 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 04:12:57 --> Class "App\Controllers\Date" not found
in APPPATH/Controllers/User.php on line 56.
 1 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 2 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 3 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 04:25:19 --> Call to undefined method CodeIgniter\Database\MySQLi\Result::num_rows()
in APPPATH/Controllers/User.php on line 59.
 1 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 2 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 3 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 05:16:50 --> Call to undefined method CodeIgniter\Database\MySQLi\Result::num_rows()
in APPPATH/Controllers/User.php on line 59.
 1 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 2 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 3 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 05:17:34 --> Call to undefined method CodeIgniter\Database\MySQLi\Result::num_rows()
in APPPATH/Controllers/User.php on line 59.
 1 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 2 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 3 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 05:19:26 --> Call to undefined method CodeIgniter\Database\MySQLi\Result::countResultAll()
in APPPATH/Controllers/User.php on line 59.
 1 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 2 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 3 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 05:20:01 --> Call to undefined method CodeIgniter\Database\MySQLi\Result::countAllResults()
in APPPATH/Controllers/User.php on line 59.
 1 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 2 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 3 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 05:20:05 --> Call to undefined method CodeIgniter\Database\MySQLi\Result::countAllResults()
in APPPATH/Controllers/User.php on line 59.
 1 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 2 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 3 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 05:20:35 --> Call to undefined method CodeIgniter\Database\MySQLi\Result::countAll()
in APPPATH/Controllers/User.php on line 59.
 1 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 2 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 3 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 05:24:49 --> Call to undefined method CodeIgniter\Database\MySQLi\Result::getResultsArray()
in APPPATH/Controllers/User.php on line 59.
 1 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 2 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 3 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 05:24:53 --> Call to undefined method CodeIgniter\Database\MySQLi\Result::getResultsArray()
in APPPATH/Controllers/User.php on line 59.
 1 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 2 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 3 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 05:25:06 --> Call to undefined method CodeIgniter\Database\MySQLi\Result::getResultsArray()
in APPPATH/Controllers/User.php on line 59.
 1 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 2 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 3 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 05:26:43 --> Call to undefined method CodeIgniter\Database\MySQLi\Result::getResultsArray()
in APPPATH/Controllers/User.php on line 59.
 1 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 2 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 3 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 05:28:12 --> Call to undefined method CodeIgniter\Database\MySQLi\Result::getResultsArray()
in APPPATH/Controllers/User.php on line 59.
 1 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 2 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 3 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 05:28:15 --> Call to undefined method CodeIgniter\Database\MySQLi\Result::getResultsArray()
in APPPATH/Controllers/User.php on line 59.
 1 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 2 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 3 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 05:54:48 --> mysqli_sql_exception: Unknown column 'created_at' in 'where clause' in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:306
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php(306): mysqli->query('SELECT * FROM `...', 0)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT * FROM `...')
#2 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT * FROM `...')
#3 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#6 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2024-01-14 05:54:48 --> Unknown column 'created_at' in 'where clause'
in SYSTEMPATH/Database/BaseConnection.php on line 647.
 1 APPPATH/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `customer` WHERE DAY(`created_at`)=14 AND MONTH(`created_at`)=01 AND YEAR(`created_at`)=2024')
 2 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 3 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 4 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 05:55:00 --> mysqli_sql_exception: Unknown column 'created_at' in 'where clause' in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:306
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php(306): mysqli->query('SELECT * FROM `...', 0)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT * FROM `...')
#2 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT * FROM `...')
#3 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#6 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2024-01-14 05:55:00 --> Unknown column 'created_at' in 'where clause'
in SYSTEMPATH/Database/BaseConnection.php on line 647.
 1 APPPATH/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `customer` WHERE DAY(`created_at`)=14 AND MONTH(`created_at`)=01 AND YEAR(`created_at`)=2024')
 2 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 3 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 4 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 08:24:28 --> Class "App\Controllers\Util" not found
in APPPATH/Controllers/User.php on line 56.
 1 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 2 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 3 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 08:24:58 --> Class "App\Controllers\Util" not found
in APPPATH/Controllers/User.php on line 56.
 1 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 2 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 3 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 08:26:19 --> Class "App\Controllers\Util" not found
in APPPATH/Controllers/User.php on line 56.
 1 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 2 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 3 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 08:26:57 --> Class "App\Controllers\Util" not found
in APPPATH/Controllers/User.php on line 56.
 1 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 2 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 3 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 08:27:12 --> Class "App\Controllers\Util" not found
in APPPATH/Controllers/User.php on line 56.
 1 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 2 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 3 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 08:27:33 --> syntax error, unexpected token "public", expecting end of file
in APPPATH/Controllers/Util.php on line 3.
 1 SYSTEMPATH/Autoloader/Autoloader.php(291): CodeIgniter\Autoloader\Autoloader->includeFile('/storage/emulated/0/htdocs/mtm_api/app/Controllers/Util.php')
 2 SYSTEMPATH/Autoloader/Autoloader.php(269): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Util')
 3 APPPATH/Controllers/User.php(56): CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Util')
 4 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 5 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 6 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 08:31:17 --> Class "App\Controllers\Util" not found
in APPPATH/Controllers/User.php on line 56.
 1 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 2 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 3 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 08:31:21 --> Class "App\Controllers\Util" not found
in APPPATH/Controllers/User.php on line 56.
 1 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 2 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 3 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 08:31:43 --> Call to undefined function App\Controllers\getDistanceDays()
in APPPATH/Controllers/User.php on line 56.
 1 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 2 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 3 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 08:32:15 --> Call to undefined function App\Controllers\getDistanceDays()
in APPPATH/Controllers/User.php on line 57.
 1 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 2 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 3 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 11:47:48 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '` AND `subscription_id`=1' at line 1 in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:306
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php(306): mysqli->query('SELECT * FROM `...', 0)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT * FROM `...')
#2 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT * FROM `...')
#3 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(101): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#6 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2024-01-14 11:47:49 --> You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '` AND `subscription_id`=1' at line 1
in SYSTEMPATH/Database/BaseConnection.php on line 647.
 1 APPPATH/Controllers/User.php(101): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `overdue` WHERE `customer_id`=1` AND `subscription_id`=1')
 2 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 3 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 4 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 11:48:01 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '` AND `subscription_id`=1' at line 1 in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:306
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php(306): mysqli->query('SELECT * FROM `...', 0)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT * FROM `...')
#2 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT * FROM `...')
#3 /storage/emulated/0/htdocs/mtm_api/app/Controllers/Test.php(15): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\Test->customer()
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Test))
#6 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2024-01-14 11:48:01 --> You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '` AND `subscription_id`=1' at line 1
in SYSTEMPATH/Database/BaseConnection.php on line 647.
 1 APPPATH/Controllers/Test.php(15): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `overdue` WHERE `customer_id`=1` AND `subscription_id`=1')
 2 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\Test->customer()
 3 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Test))
 4 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 11:48:04 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '` AND `subscription_id`=1' at line 1 in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:306
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php(306): mysqli->query('SELECT * FROM `...', 0)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT * FROM `...')
#2 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT * FROM `...')
#3 /storage/emulated/0/htdocs/mtm_api/app/Controllers/Test.php(15): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\Test->customer()
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Test))
#6 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2024-01-14 11:48:04 --> You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '` AND `subscription_id`=1' at line 1
in SYSTEMPATH/Database/BaseConnection.php on line 647.
 1 APPPATH/Controllers/Test.php(15): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `overdue` WHERE `customer_id`=1` AND `subscription_id`=1')
 2 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\Test->customer()
 3 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Test))
 4 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 11:51:07 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '` AND `subscription_id`=1' at line 1 in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:306
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php(306): mysqli->query('SELECT * FROM `...', 0)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT * FROM `...')
#2 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT * FROM `...')
#3 /storage/emulated/0/htdocs/mtm_api/app/Controllers/Test.php(15): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\Test->customer()
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Test))
#6 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2024-01-14 11:51:07 --> You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '` AND `subscription_id`=1' at line 1
in SYSTEMPATH/Database/BaseConnection.php on line 647.
 1 APPPATH/Controllers/Test.php(15): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `overdue` WHERE `customer_id`=1` AND `subscription_id`=1')
 2 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\Test->customer()
 3 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Test))
 4 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 11:54:25 --> Undefined variable $date
in APPPATH/Controllers/Test.php on line 18.
 1 APPPATH/Controllers/Test.php(18): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $date', '/storage/emulated/0/htdocs/mtm_api/app/Controllers/Test.php', 18)
 2 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\Test->customer()
 3 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Test))
 4 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 11:58:56 --> Undefined variable $disabledCustomers
in APPPATH/Controllers/Test.php on line 21.
 1 APPPATH/Controllers/Test.php(21): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $disabledCustomers', '/storage/emulated/0/htdocs/mtm_api/app/Controllers/Test.php', 21)
 2 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\Test->customer()
 3 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Test))
 4 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 11:59:18 --> Undefined variable $disabledCustomers
in APPPATH/Controllers/Test.php on line 21.
 1 APPPATH/Controllers/Test.php(21): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $disabledCustomers', '/storage/emulated/0/htdocs/mtm_api/app/Controllers/Test.php', 21)
 2 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\Test->customer()
 3 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Test))
 4 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 11:59:28 --> Undefined variable $disabledCustomers
in APPPATH/Controllers/Test.php on line 27.
 1 APPPATH/Controllers/Test.php(27): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $disabledCustomers', '/storage/emulated/0/htdocs/mtm_api/app/Controllers/Test.php', 27)
 2 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\Test->customer()
 3 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Test))
 4 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 11:59:58 --> Undefined variable $disabledCustomers
in APPPATH/Controllers/Test.php on line 22.
 1 APPPATH/Controllers/Test.php(22): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $disabledCustomers', '/storage/emulated/0/htdocs/mtm_api/app/Controllers/Test.php', 22)
 2 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\Test->customer()
 3 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Test))
 4 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 12:00:24 --> Undefined variable $disabledCustomers
in APPPATH/Controllers/Test.php on line 23.
 1 APPPATH/Controllers/Test.php(23): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $disabledCustomers', '/storage/emulated/0/htdocs/mtm_api/app/Controllers/Test.php', 23)
 2 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\Test->customer()
 3 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Test))
 4 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 12:24:22 --> mysqli_sql_exception: Unknown column 'date' in 'where clause' in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:306
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php(306): mysqli->query('SELECT * FROM `...', 0)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT * FROM `...')
#2 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT * FROM `...')
#3 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(70): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#6 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2024-01-14 12:24:22 --> Unknown column 'date' in 'where clause'
in SYSTEMPATH/Database/BaseConnection.php on line 647.
 1 APPPATH/Controllers/User.php(70): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `overdue` WHERE DAY(`date`)=14 AND MONTH(`date`)=01 AND YEAR(`date`)=2024 AND `paid`=0')
 2 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 3 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 4 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 12:24:23 --> mysqli_sql_exception: Unknown column 'date' in 'where clause' in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:306
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php(306): mysqli->query('SELECT * FROM `...', 0)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT * FROM `...')
#2 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT * FROM `...')
#3 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(70): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#6 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2024-01-14 12:24:24 --> Unknown column 'date' in 'where clause'
in SYSTEMPATH/Database/BaseConnection.php on line 647.
 1 APPPATH/Controllers/User.php(70): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `overdue` WHERE DAY(`date`)=14 AND MONTH(`date`)=01 AND YEAR(`date`)=2024 AND `paid`=0')
 2 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 3 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 4 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 12:25:12 --> mysqli_sql_exception: Unknown column 'date' in 'where clause' in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:306
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php(306): mysqli->query('SELECT * FROM `...', 0)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT * FROM `...')
#2 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT * FROM `...')
#3 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(70): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#6 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2024-01-14 12:25:13 --> Unknown column 'date' in 'where clause'
in SYSTEMPATH/Database/BaseConnection.php on line 647.
 1 APPPATH/Controllers/User.php(70): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `overdue` WHERE DAY(`date`)=14 AND MONTH(`date`)=01 AND YEAR(`date`)=2024 AND `paid`=0')
 2 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 3 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 4 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 12:25:40 --> mysqli_sql_exception: Unknown column 'date' in 'where clause' in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:306
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php(306): mysqli->query('SELECT * FROM `...', 0)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT * FROM `...')
#2 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT * FROM `...')
#3 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(70): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#6 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2024-01-14 12:25:40 --> Unknown column 'date' in 'where clause'
in SYSTEMPATH/Database/BaseConnection.php on line 647.
 1 APPPATH/Controllers/User.php(70): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `overdue` WHERE DAY(`date`)=14 AND MONTH(`date`)=01 AND YEAR(`date`)=2024 AND `paid`=0')
 2 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 3 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 4 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 12:27:27 --> mysqli_sql_exception: Unknown column 'date' in 'where clause' in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:306
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php(306): mysqli->query('SELECT * FROM `...', 0)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT * FROM `...')
#2 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT * FROM `...')
#3 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(70): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#6 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2024-01-14 12:27:27 --> Unknown column 'date' in 'where clause'
in SYSTEMPATH/Database/BaseConnection.php on line 647.
 1 APPPATH/Controllers/User.php(70): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `overdue` WHERE DAY(`date`)=14 AND MONTH(`date`)=01 AND YEAR(`date`)=2024 AND `paid`=0')
 2 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 3 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 4 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 12:27:38 --> mysqli_sql_exception: Unknown column 'date' in 'where clause' in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:306
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php(306): mysqli->query('SELECT * FROM `...', 0)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT * FROM `...')
#2 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT * FROM `...')
#3 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(70): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#6 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2024-01-14 12:27:38 --> Unknown column 'date' in 'where clause'
in SYSTEMPATH/Database/BaseConnection.php on line 647.
 1 APPPATH/Controllers/User.php(70): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `overdue` WHERE DAY(`date`)=14 AND MONTH(`date`)=01 AND YEAR(`date`)=2024 AND `paid`=0')
 2 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 3 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 4 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 12:28:15 --> mysqli_sql_exception: Unknown column 'date' in 'where clause' in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:306
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php(306): mysqli->query('SELECT * FROM `...', 0)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT * FROM `...')
#2 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT * FROM `...')
#3 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(70): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#6 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2024-01-14 12:28:15 --> Unknown column 'date' in 'where clause'
in SYSTEMPATH/Database/BaseConnection.php on line 647.
 1 APPPATH/Controllers/User.php(70): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `overdue` WHERE DAY(`date`)=14 AND MONTH(`date`)=01 AND YEAR(`date`)=2024 AND `paid`=0')
 2 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 3 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 4 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 12:28:54 --> mysqli_sql_exception: Unknown column 'date' in 'where clause' in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:306
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php(306): mysqli->query('SELECT * FROM `...', 0)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT * FROM `...')
#2 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT * FROM `...')
#3 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(76): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#6 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2024-01-14 12:28:54 --> Unknown column 'date' in 'where clause'
in SYSTEMPATH/Database/BaseConnection.php on line 647.
 1 APPPATH/Controllers/User.php(76): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `overdue` WHERE DAY(`date`)=14 AND MONTH(`date`)=01 AND YEAR(`date`)=2024 AND `paid`=`overdue`')
 2 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 3 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 4 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 12:28:57 --> mysqli_sql_exception: Unknown column 'date' in 'where clause' in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:306
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php(306): mysqli->query('SELECT * FROM `...', 0)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT * FROM `...')
#2 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT * FROM `...')
#3 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(76): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#6 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2024-01-14 12:28:57 --> Unknown column 'date' in 'where clause'
in SYSTEMPATH/Database/BaseConnection.php on line 647.
 1 APPPATH/Controllers/User.php(76): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `overdue` WHERE DAY(`date`)=14 AND MONTH(`date`)=01 AND YEAR(`date`)=2024 AND `paid`=`overdue`')
 2 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 3 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 4 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 12:43:07 --> syntax error, unexpected token "=>", expecting ")"
in APPPATH/Controllers/User.php on line 141.
 1 SYSTEMPATH/Autoloader/Autoloader.php(291): CodeIgniter\Autoloader\Autoloader->includeFile('/storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php')
 2 SYSTEMPATH/Autoloader/Autoloader.php(269): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\User')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\User')
 4 SYSTEMPATH/CodeIgniter.php(900): class_exists('\\App\\Controllers\\User', true)
 5 SYSTEMPATH/CodeIgniter.php(489): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 12:46:21 --> syntax error, unexpected token "=>", expecting ")"
in APPPATH/Controllers/User.php on line 145.
 1 SYSTEMPATH/Autoloader/Autoloader.php(291): CodeIgniter\Autoloader\Autoloader->includeFile('/storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php')
 2 SYSTEMPATH/Autoloader/Autoloader.php(269): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\User')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\User')
 4 SYSTEMPATH/CodeIgniter.php(900): class_exists('\\App\\Controllers\\User', true)
 5 SYSTEMPATH/CodeIgniter.php(489): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 12:46:26 --> syntax error, unexpected token "=>", expecting ")"
in APPPATH/Controllers/User.php on line 145.
 1 SYSTEMPATH/Autoloader/Autoloader.php(291): CodeIgniter\Autoloader\Autoloader->includeFile('/storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php')
 2 SYSTEMPATH/Autoloader/Autoloader.php(269): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\User')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\User')
 4 SYSTEMPATH/CodeIgniter.php(900): class_exists('\\App\\Controllers\\User', true)
 5 SYSTEMPATH/CodeIgniter.php(489): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 12:46:26 --> syntax error, unexpected token "=>", expecting ")"
in APPPATH/Controllers/User.php on line 145.
 1 SYSTEMPATH/Autoloader/Autoloader.php(291): CodeIgniter\Autoloader\Autoloader->includeFile('/storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php')
 2 SYSTEMPATH/Autoloader/Autoloader.php(269): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\User')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\User')
 4 SYSTEMPATH/CodeIgniter.php(900): class_exists('\\App\\Controllers\\User', true)
 5 SYSTEMPATH/CodeIgniter.php(489): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 12:46:27 --> syntax error, unexpected token "=>", expecting ")"
in APPPATH/Controllers/User.php on line 145.
 1 SYSTEMPATH/Autoloader/Autoloader.php(291): CodeIgniter\Autoloader\Autoloader->includeFile('/storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php')
 2 SYSTEMPATH/Autoloader/Autoloader.php(269): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\User')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\User')
 4 SYSTEMPATH/CodeIgniter.php(900): class_exists('\\App\\Controllers\\User', true)
 5 SYSTEMPATH/CodeIgniter.php(489): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 12:46:27 --> syntax error, unexpected token "=>", expecting ")"
in APPPATH/Controllers/User.php on line 145.
 1 SYSTEMPATH/Autoloader/Autoloader.php(291): CodeIgniter\Autoloader\Autoloader->includeFile('/storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php')
 2 SYSTEMPATH/Autoloader/Autoloader.php(269): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\User')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\User')
 4 SYSTEMPATH/CodeIgniter.php(900): class_exists('\\App\\Controllers\\User', true)
 5 SYSTEMPATH/CodeIgniter.php(489): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 12:46:48 --> syntax error, unexpected token "=>", expecting ")"
in APPPATH/Controllers/User.php on line 145.
 1 SYSTEMPATH/Autoloader/Autoloader.php(291): CodeIgniter\Autoloader\Autoloader->includeFile('/storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php')
 2 SYSTEMPATH/Autoloader/Autoloader.php(269): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\User')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\User')
 4 SYSTEMPATH/CodeIgniter.php(900): class_exists('\\App\\Controllers\\User', true)
 5 SYSTEMPATH/CodeIgniter.php(489): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 12:47:33 --> syntax error, unexpected token "=>", expecting ")"
in APPPATH/Controllers/User.php on line 146.
 1 SYSTEMPATH/Autoloader/Autoloader.php(291): CodeIgniter\Autoloader\Autoloader->includeFile('/storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php')
 2 SYSTEMPATH/Autoloader/Autoloader.php(269): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\User')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\User')
 4 SYSTEMPATH/CodeIgniter.php(900): class_exists('\\App\\Controllers\\User', true)
 5 SYSTEMPATH/CodeIgniter.php(489): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 12:48:01 --> syntax error, unexpected token "=>", expecting ")"
in APPPATH/Controllers/User.php on line 147.
 1 SYSTEMPATH/Autoloader/Autoloader.php(291): CodeIgniter\Autoloader\Autoloader->includeFile('/storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php')
 2 SYSTEMPATH/Autoloader/Autoloader.php(269): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\User')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\User')
 4 SYSTEMPATH/CodeIgniter.php(900): class_exists('\\App\\Controllers\\User', true)
 5 SYSTEMPATH/CodeIgniter.php(489): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 12:48:34 --> syntax error, unexpected token "=>", expecting ")"
in APPPATH/Controllers/User.php on line 147.
 1 SYSTEMPATH/Autoloader/Autoloader.php(291): CodeIgniter\Autoloader\Autoloader->includeFile('/storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php')
 2 SYSTEMPATH/Autoloader/Autoloader.php(269): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\User')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\User')
 4 SYSTEMPATH/CodeIgniter.php(900): class_exists('\\App\\Controllers\\User', true)
 5 SYSTEMPATH/CodeIgniter.php(489): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 12:48:59 --> syntax error, unexpected token "=>", expecting ")"
in APPPATH/Controllers/User.php on line 147.
 1 SYSTEMPATH/Autoloader/Autoloader.php(291): CodeIgniter\Autoloader\Autoloader->includeFile('/storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php')
 2 SYSTEMPATH/Autoloader/Autoloader.php(269): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\User')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\User')
 4 SYSTEMPATH/CodeIgniter.php(900): class_exists('\\App\\Controllers\\User', true)
 5 SYSTEMPATH/CodeIgniter.php(489): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 12:50:05 --> syntax error, unexpected token "=>", expecting ")"
in APPPATH/Controllers/User.php on line 147.
 1 SYSTEMPATH/Autoloader/Autoloader.php(291): CodeIgniter\Autoloader\Autoloader->includeFile('/storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php')
 2 SYSTEMPATH/Autoloader/Autoloader.php(269): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\User')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\User')
 4 SYSTEMPATH/CodeIgniter.php(900): class_exists('\\App\\Controllers\\User', true)
 5 SYSTEMPATH/CodeIgniter.php(489): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 12:59:53 --> Error connecting to the database: mysqli_sql_exception: MySQL server has gone away in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:184
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php(184): mysqli->real_connect('localhost', 'mtm', Object(SensitiveParameterValue), 'mtm', 3306, '', 0)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(143): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_unpaying_customers()
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#6 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: MySQL server has gone away in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:229
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(143): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_unpaying_customers()
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2024-01-14 12:59:53 --> Error connecting to the database: ErrorException: mysqli::real_connect(): Error while reading greeting packet. PID=20872 in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:184
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', '/storage/emulat...', 184)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php(184): mysqli->real_connect('localhost', 'mtm', Object(SensitiveParameterValue), 'mtm', 3306, '', 0)
#2 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#4 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
#6 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#7 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}

Next mysqli_sql_exception: MySQL server has gone away in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:184
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php(184): mysqli->real_connect('localhost', 'mtm', Object(SensitiveParameterValue), 'mtm', 3306, '', 0)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#6 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: MySQL server has gone away in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:229
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2024-01-14 12:59:53 --> Unable to connect to the database.
Main connection [MySQLi]: MySQL server has gone away
in SYSTEMPATH/Database/BaseConnection.php on line 428.
 1 SYSTEMPATH/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH/Controllers/User.php(143): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `overdue` WHERE `paid`=0 AND MONTH(`updated_at`)=01 AND YEAR(`updated_at`)=2024')
 3 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_unpaying_customers()
 4 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 12:59:53 --> Unable to connect to the database.
Main connection [MySQLi]: MySQL server has gone away
in SYSTEMPATH/Database/BaseConnection.php on line 428.
 1 SYSTEMPATH/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `customer`')
 3 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 4 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 12:59:54 --> Error connecting to the database: mysqli_sql_exception: Connection refused in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:184
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php(184): mysqli->real_connect('localhost', 'mtm', Object(SensitiveParameterValue), 'mtm', 3306, '', 0)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(143): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_unpaying_customers()
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#6 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Connection refused in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:229
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(143): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_unpaying_customers()
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2024-01-14 12:59:54 --> Error connecting to the database: mysqli_sql_exception: Connection refused in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:184
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php(184): mysqli->real_connect('localhost', 'mtm', Object(SensitiveParameterValue), 'mtm', 3306, '', 0)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#6 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Connection refused in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:229
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2024-01-14 12:59:54 --> Unable to connect to the database.
Main connection [MySQLi]: Connection refused
in SYSTEMPATH/Database/BaseConnection.php on line 428.
 1 SYSTEMPATH/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH/Controllers/User.php(143): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `overdue` WHERE `paid`=0 AND MONTH(`updated_at`)=01 AND YEAR(`updated_at`)=2024')
 3 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_unpaying_customers()
 4 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 12:59:54 --> Unable to connect to the database.
Main connection [MySQLi]: Connection refused
in SYSTEMPATH/Database/BaseConnection.php on line 428.
 1 SYSTEMPATH/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `customer`')
 3 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 4 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 12:59:59 --> Error connecting to the database: mysqli_sql_exception: Connection refused in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:184
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php(184): mysqli->real_connect('localhost', 'mtm', Object(SensitiveParameterValue), 'mtm', 3306, '', 0)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#6 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Connection refused in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:229
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2024-01-14 13:00:00 --> Unable to connect to the database.
Main connection [MySQLi]: Connection refused
in SYSTEMPATH/Database/BaseConnection.php on line 428.
 1 SYSTEMPATH/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `customer`')
 3 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 4 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 13:00:00 --> Error connecting to the database: mysqli_sql_exception: Connection refused in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:184
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php(184): mysqli->real_connect('localhost', 'mtm', Object(SensitiveParameterValue), 'mtm', 3306, '', 0)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(143): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_unpaying_customers()
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#6 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Connection refused in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:229
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(143): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_unpaying_customers()
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2024-01-14 13:00:00 --> Unable to connect to the database.
Main connection [MySQLi]: Connection refused
in SYSTEMPATH/Database/BaseConnection.php on line 428.
 1 SYSTEMPATH/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH/Controllers/User.php(143): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `overdue` WHERE `paid`=0 AND MONTH(`updated_at`)=01 AND YEAR(`updated_at`)=2024')
 3 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_unpaying_customers()
 4 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 13:00:16 --> Error connecting to the database: mysqli_sql_exception: Connection refused in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:184
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php(184): mysqli->real_connect('localhost', 'mtm', Object(SensitiveParameterValue), 'mtm', 3306, '', 0)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(143): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_unpaying_customers()
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#6 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Connection refused in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:229
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(143): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_unpaying_customers()
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2024-01-14 13:00:16 --> Unable to connect to the database.
Main connection [MySQLi]: Connection refused
in SYSTEMPATH/Database/BaseConnection.php on line 428.
 1 SYSTEMPATH/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH/Controllers/User.php(143): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `overdue` WHERE `paid`=0 AND MONTH(`updated_at`)=01 AND YEAR(`updated_at`)=2024')
 3 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_unpaying_customers()
 4 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 13:00:16 --> Error connecting to the database: mysqli_sql_exception: Connection refused in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:184
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php(184): mysqli->real_connect('localhost', 'mtm', Object(SensitiveParameterValue), 'mtm', 3306, '', 0)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#6 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Connection refused in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:229
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2024-01-14 13:00:16 --> Unable to connect to the database.
Main connection [MySQLi]: Connection refused
in SYSTEMPATH/Database/BaseConnection.php on line 428.
 1 SYSTEMPATH/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `customer`')
 3 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 4 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 13:00:45 --> Error connecting to the database: mysqli_sql_exception: Connection refused in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:184
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php(184): mysqli->real_connect('localhost', 'mtm', Object(SensitiveParameterValue), 'mtm', 3306, '', 0)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#6 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Connection refused in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:229
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2024-01-14 13:00:45 --> Unable to connect to the database.
Main connection [MySQLi]: Connection refused
in SYSTEMPATH/Database/BaseConnection.php on line 428.
 1 SYSTEMPATH/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `customer`')
 3 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 4 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 13:00:45 --> Error connecting to the database: mysqli_sql_exception: Connection refused in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:184
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php(184): mysqli->real_connect('localhost', 'mtm', Object(SensitiveParameterValue), 'mtm', 3306, '', 0)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(143): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_unpaying_customers()
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#6 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Connection refused in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:229
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(143): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_unpaying_customers()
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2024-01-14 13:00:45 --> Unable to connect to the database.
Main connection [MySQLi]: Connection refused
in SYSTEMPATH/Database/BaseConnection.php on line 428.
 1 SYSTEMPATH/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH/Controllers/User.php(143): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `overdue` WHERE `paid`=0 AND MONTH(`updated_at`)=01 AND YEAR(`updated_at`)=2024')
 3 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_unpaying_customers()
 4 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 13:03:08 --> Error connecting to the database: mysqli_sql_exception: Connection refused in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:184
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php(184): mysqli->real_connect('localhost', 'mtm', Object(SensitiveParameterValue), 'mtm', 3306, '', 0)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#6 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Connection refused in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:229
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2024-01-14 13:03:08 --> Error connecting to the database: mysqli_sql_exception: Connection refused in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:184
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php(184): mysqli->real_connect('localhost', 'mtm', Object(SensitiveParameterValue), 'mtm', 3306, '', 0)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(143): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_unpaying_customers()
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#6 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Connection refused in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:229
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(143): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_unpaying_customers()
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2024-01-14 13:03:08 --> Unable to connect to the database.
Main connection [MySQLi]: Connection refused
in SYSTEMPATH/Database/BaseConnection.php on line 428.
 1 SYSTEMPATH/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH/Controllers/User.php(143): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `overdue` WHERE `paid`=0 AND MONTH(`updated_at`)=01 AND YEAR(`updated_at`)=2024')
 3 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_unpaying_customers()
 4 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 13:03:08 --> Unable to connect to the database.
Main connection [MySQLi]: Connection refused
in SYSTEMPATH/Database/BaseConnection.php on line 428.
 1 SYSTEMPATH/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `customer`')
 3 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 4 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 13:04:02 --> Error connecting to the database: mysqli_sql_exception: Connection refused in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:184
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php(184): mysqli->real_connect('localhost', 'mtm', Object(SensitiveParameterValue), 'mtm', 3306, '', 0)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(143): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_unpaying_customers()
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#6 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Connection refused in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:229
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(143): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_unpaying_customers()
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2024-01-14 13:04:02 --> Unable to connect to the database.
Main connection [MySQLi]: Connection refused
in SYSTEMPATH/Database/BaseConnection.php on line 428.
 1 SYSTEMPATH/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH/Controllers/User.php(143): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `overdue` WHERE `paid`=0 AND MONTH(`updated_at`)=01 AND YEAR(`updated_at`)=2024')
 3 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_unpaying_customers()
 4 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 13:04:02 --> Error connecting to the database: mysqli_sql_exception: Connection refused in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:184
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php(184): mysqli->real_connect('localhost', 'mtm', Object(SensitiveParameterValue), 'mtm', 3306, '', 0)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#6 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Connection refused in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:229
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2024-01-14 13:04:02 --> Unable to connect to the database.
Main connection [MySQLi]: Connection refused
in SYSTEMPATH/Database/BaseConnection.php on line 428.
 1 SYSTEMPATH/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `customer`')
 3 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 4 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 13:04:31 --> Error connecting to the database: mysqli_sql_exception: Connection refused in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:184
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php(184): mysqli->real_connect('localhost', 'mtm', Object(SensitiveParameterValue), 'mtm', 3306, '', 0)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(143): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_unpaying_customers()
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#6 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Connection refused in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:229
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(143): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_unpaying_customers()
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2024-01-14 13:04:31 --> Unable to connect to the database.
Main connection [MySQLi]: Connection refused
in SYSTEMPATH/Database/BaseConnection.php on line 428.
 1 SYSTEMPATH/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH/Controllers/User.php(143): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `overdue` WHERE `paid`=0 AND MONTH(`updated_at`)=01 AND YEAR(`updated_at`)=2024')
 3 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_unpaying_customers()
 4 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 13:04:40 --> Error connecting to the database: mysqli_sql_exception: Connection refused in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:184
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php(184): mysqli->real_connect('localhost', 'mtm', Object(SensitiveParameterValue), 'mtm', 3306, '', 0)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#6 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Connection refused in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:229
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2024-01-14 13:04:40 --> Unable to connect to the database.
Main connection [MySQLi]: Connection refused
in SYSTEMPATH/Database/BaseConnection.php on line 428.
 1 SYSTEMPATH/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `customer`')
 3 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 4 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 13:05:32 --> Error connecting to the database: mysqli_sql_exception: Connection refused in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:184
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php(184): mysqli->real_connect('localhost', 'mtm', Object(SensitiveParameterValue), 'mtm', 3306, '', 0)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#6 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Connection refused in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:229
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2024-01-14 13:05:32 --> Unable to connect to the database.
Main connection [MySQLi]: Connection refused
in SYSTEMPATH/Database/BaseConnection.php on line 428.
 1 SYSTEMPATH/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `customer`')
 3 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 4 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 13:05:45 --> Error connecting to the database: mysqli_sql_exception: Connection refused in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:184
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php(184): mysqli->real_connect('localhost', 'mtm', Object(SensitiveParameterValue), 'mtm', 3306, '', 0)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#6 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Connection refused in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:229
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2024-01-14 13:05:45 --> Unable to connect to the database.
Main connection [MySQLi]: Connection refused
in SYSTEMPATH/Database/BaseConnection.php on line 428.
 1 SYSTEMPATH/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `customer`')
 3 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 4 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 13:06:06 --> Error connecting to the database: mysqli_sql_exception: Connection refused in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:184
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php(184): mysqli->real_connect('localhost', 'mtm', Object(SensitiveParameterValue), 'mtm', 3306, '', 0)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#6 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Connection refused in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:229
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2024-01-14 13:06:06 --> Unable to connect to the database.
Main connection [MySQLi]: Connection refused
in SYSTEMPATH/Database/BaseConnection.php on line 428.
 1 SYSTEMPATH/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `customer`')
 3 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 4 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 13:06:28 --> Error connecting to the database: mysqli_sql_exception: Connection refused in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:184
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php(184): mysqli->real_connect('localhost', 'mtm', Object(SensitiveParameterValue), 'mtm', 3306, '', 0)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#6 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Connection refused in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:229
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2024-01-14 13:06:28 --> Unable to connect to the database.
Main connection [MySQLi]: Connection refused
in SYSTEMPATH/Database/BaseConnection.php on line 428.
 1 SYSTEMPATH/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `customer`')
 3 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 4 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 13:06:50 --> Error connecting to the database: mysqli_sql_exception: Connection refused in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:184
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php(184): mysqli->real_connect('localhost', 'mtm', Object(SensitiveParameterValue), 'mtm', 3306, '', 0)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#6 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Connection refused in /storage/emulated/0/htdocs/mtm_api/system/Database/MySQLi/Connection.php:229
Stack trace:
#0 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /storage/emulated/0/htdocs/mtm_api/system/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 /storage/emulated/0/htdocs/mtm_api/app/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
#4 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#5 /storage/emulated/0/htdocs/mtm_api/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /storage/emulated/0/htdocs/mtm_api/public/index.php(79): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2024-01-14 13:06:50 --> Unable to connect to the database.
Main connection [MySQLi]: Connection refused
in SYSTEMPATH/Database/BaseConnection.php on line 428.
 1 SYSTEMPATH/Database/BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH/Controllers/User.php(60): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `customer`')
 3 SYSTEMPATH/CodeIgniter.php(942): App\Controllers\User->get_customer_data()
 4 SYSTEMPATH/CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH/CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH/index.php(79): CodeIgniter\CodeIgniter->run()
